dApp Staking V3 Dev Guide (#582)
AstarNetwork/astar-docs: docs/use/dapp-staking/for-devs/requirements.md

- After the voting is passed, you are can follow the registration instructions in the [Register dApp](/docs/use/dapp-staking/for-devs/register-dapp.md) page.

#### What about other projects who can't meet the above requirements?

You can't add your project to DefilLama because it's not generating any TVL, also DappRadar is not an option because there is no use of smart contracts? These are the current steps to follow:

- Be active in our community Discord channel and post your project updates in our dedicated `#project discussion` channel.
- Create a proposal to join dApp staking on our [forum](https://forum.astar.network/c/initiatives/dapp-staking-applications/21). For creating your proposal, you can take inspiration from the following [template](/docs/use/dapp-staking/for-devs/requirements#template-for-dapp-staking-application).
- After the proposal is created, we will get the community to discuss. You are expected to answer all the questions. This will take around 4-5 days.
- The voting will be opened after the discussion period.
- For now, only the forum members with Trust Level 2 and above are allowed to vote.

> - After the voting is passed, you are can follow the registration instructions in the [Register dApp](/docs/use/dapp-staking/for-devs/register-dapp.md) page.

Reviewer: ```suggestion
- After the voting is passed, you should follow the registration instructions in the [Register dApp](/docs/use/dapp-staking/for-devs/register-dapp.md) page.
```
